Stack/Opc.Ua.Core/Stack/Tcp/UaSCBinaryChannel.Asymmetric.cs

```diff
@@ -195,17 +182,149 @@ protected static void CompareCertificates(X509Certificate2 expected, X509Certifi
         /// </summary>
         protected uint GetNonceLength()
         {
-            return Utils.Nonce.GetNonceLength(SecurityPolicyUri);
+            switch (SecurityPolicyUri)
```

Comment:
as I mentioned before, maybe we can retire the nonce length setting from the config files...